source/class/qx/bom/Label.js

```diff
@@ -304,6 +316,13 @@ qx.Bootstrap.define("qx.bom.Label", {
      * @return {Map} A map with preferred <code>width</code> and <code>height</code>.
      */
     getTextSize(text, styles) {
+      let cacheKey = this.__getCacheKey(styles);
+      let cache = qx.bom.Label.__sizeCache[cacheKey];
+      let size = cache?.sizes[text];
```

Comment:
`__sizeCache` is opaque, i.e., manipulated only by its accessor functions, everyplace except here. Suggest replacing this reference to the internal `sizes` with

```
let size = __getCachedSize(cacheKey, text);
```

Reply:
done